internal/report/report.go

```diff
@@ -1224,13 +1239,19 @@ func computeTotal(prof *profile.Profile, value, meanDiv func(v []int64) int64) i
 		if v < 0 {
 			v = -v
 		}
-		ret += v
+		total += v
+		if sample.HasTag("pprof::diff", "true") {
+			diffTotal += v
+		}
 		div += d
```

Comment:
Assuming that it's intentional that accumulating the div is not under the tag condition, can you add a comment on why so?

Reply:
fixed.